Automattic/pocket-casts-android
# Create NumberStepper component in Compose UI and replace it for Sleep Timer

## Description
- This migrates the stepper from sleep timer to Compose UI and creates it as a component

## Testing Instructions
1. Play an episode
2. Open Sleep timer
3. Make sure you can update the `plus` and `minor` buttons ✅



## Screenshots or Screencast 
| Before | After |
|--------|--------|
| ![before](https://github.com/user-attachments/assets/e8cb0c0c-61fa-4993-b710-c7063209c89b) | ![after](https://github.com/user-attachments/assets/4c31077b-f3cb-4e27-992f-d01925f1b1cb) | 


## Checklist
 
#### I have tested any UI changes...
